### DISCUSSION TITLE
[question] 请问@antv/g6-react-node 和 @antv/g6-extension-react 的差别是什么？

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#6468] [Q&A] [answered]
请问：
[@antv/g6-react-node ](https://www.npmjs.com/package/@antv/g6-react-node)和 [@antv/g6-extension-react ](https://www.npmjs.com/package/@antv/g6-extension-react)的差别是什么呢？

**Comment by liaodalin19903** [comment, marked as answer]
v5开始直接使用：@antv/g6-extension-react   即可。

**Reply by hustcc** [reply]
后面 antv 会慢慢形成规范，扩展都以 extension 来。@antv/g6-react-node  应该是给 v4 的 react 来用的，不是特别的规范。
